scipy/io/_fast_matrix_market/__init__.py

```diff
@@ -293,20 +294,29 @@ def _validate_symmetry(symmetry):
     return symmetry
 
 
-def mmread(source):
+def mmread(source, sparray=None):
```

Comment:
Can we make the new arguments kwarg only
```suggestion
def mmread(source, *, sparray=None):
```